src/openms/source/ANALYSIS/ID/FalseDiscoveryRate.cpp

```diff
@@ -505,7 +505,7 @@ namespace OpenMS
           new_hits.push_back(std::move(hit));
         }
       }
-      it->setHits(std::move(new_hits));
+      it->setHits(new_hits);
```

Comment:
we don't have setHits with &&

Reply:
instead of removing this we should implement it
```suggestion
      it->setHits(std::move(new_hits)); // TODO implement setHits &&
```

Reply:
totally agree

Reply:
how can we implement that?
    /// Sets the peptide hits
    void setHits(std::vector<PeptideHit>&& hits);
doesn't work with pyOpenMS
and having both will result in "cannot be overloaded".
But I thought that only applies to templates https://stackoverflow.com/questions/27835925/overload-between-rvalue-reference-and-const-lvalue-reference-in-template

Reply:
will address this is in  https://github.com/OpenMS/OpenMS/pull/5685